tensorboard/uploader/uploader_main.py

```diff
@@ -409,6 +409,8 @@ def execute(self, server_info, channel):
             url = server_info_lib.experiment_url(server_info, experiment_id)
             print(url)
             data = [
+                ("Name", experiment.name),
+                ("Description", experiment.description),
```

Comment:
The description says that “OSS product users should see no chnage”, but
this is a user-facing change; `tensorboard dev list` now shows
additional `Name` and `Description` fields, with empty values:

```
$ tensorboard dev list
https://tensorboard.dev/experiment/AdYd1TgeTlaLWXx6I8JUbA/
	Name       
	Description 
	Id         AdYd1TgeTlaLWXx6I8JUbA
    ...
```

Is this intended?